### ISSUE TITLE
all blocks between expression

### ISSUE THREAD

**Issue by Runakai1** [SkriptLang/Skript#3021]
Skript:
 2.5alpha3
Paperspigot 1.15.2

The expression doesn't work. Did multiple testing. No console errors, just returns none

```command blocks:
	trigger:
		set {_loc1} to location of player
		set {_loc2} to {_loc1}
		remove 7 from x-coordinate of {_loc2}
		remove 3 from z-coordinate of {_loc2}
		set {_list::*} to all blocks between {_loc1} and {_loc2}
		send "%{_list::*}%"
		set {_a::*} to all blocks between player and block 10 meters in front of player
		send "%{_a::*}%"

**Comment by Runakai1** [comment]
Is there a fix yet?

**Comment by APickledWalrus** [comment]
I’ll do some testing on this one and report back.

**Comment by ShaneBeee** [comment]
Marking this as an enhancement
The current syntax is `blocks within %block% and %block%` that is why its not working for you. You are using locations not blocks.

Im doing a PR which will use locations instead.

**Comment by TPGamesNL** [comment]
Should be fixed in #3320
